### ISSUE TITLE
Add validation set to training set such that hyperparameters could be evaluated not on test set

### ISSUE THREAD

**Issue by weisske** [Speedfan94/PDS_Project#108]
Create Validation set from training set as 20% of whole dataset.
